webapp/src/containers/WalletPage/components/SendPage/index.tsx

```diff
@@ -292,21 +285,23 @@ class SendPage extends Component<SendPageProps, SendPageState> {
       regularDFI,
     });
     if (isAmountValid && isAddressValid) {
-      let amount;
+      let amount: BigNumber;
       let txHash;
       if (
         (!this.tokenSymbol || this.tokenSymbol === 'DFI') &&
-        regularDFI !== 0
+        !regularDFI.isZero()
       ) {
         // Convert to base unit
-        amount = getAmountInSelectedUnit(
+        amount = convertAmountFromUnit(
           this.state.amountToSendDisplayed,
           DEFAULT_UNIT,
           this.props.unit
         );
+        // amount.is
+        const feeCheck = amount.eq(this.props.sendData.walletBalance);
```

Comment:
use `gte` just in case it's safer

Reply:
Yeah, We have that check on the frontend that it needs to be greater than the amount or equal. But yeah good suggestion.